extensions/wikia/AdEngine/js/run/desktop.run.js

```diff
@@ -17,7 +17,8 @@ require([
 	'ext.wikia.adEngine.sourcePointDetection',
 	'wikia.krux',
 	'wikia.window',
-	'wikia.loader'
+	'wikia.loader',
+	require.optional('ext.wikia.adEngine.recovery.gcs')
```

Comment:
@hakubo is it still OK to use `require.optional` or you're very soon to merge Ember's `loader.js`?